app/mailers/change_request_mailer.rb

```diff
@@ -38,10 +39,20 @@ def accept(change_request, owner, url)
     @change_request = change_request
     @url = url
     @owner = owner
-
+    @email_needs_to_be_simplified = simplify_email(change_request)
     mail(
       bcc: [@change_request.requestor.email, owner.email],
       subject: "NBGallery change request accepted"
     )
   end
+
+  def simplify_email(change_request)
```

Comment:
Why are you putting control logic in a view? Because that is how this code reads. Also, the name if this function implies that it is simplifying the email, not checking if it has to be simplified.

Reply:
@mcrutch does this work? https://github.com/nbgallery/nbgallery/pull/661/commits/a14df830f22652cb4a59eec97dc7e3564d11193a

Reply:
You are still using a (currently non-existent) view to make execution flow decisions, and there is no sample or documentation for how it would even work either.  Why can't this be a function or functions in the subscription controller than can be overwritten by an extension? Similar to how all the other custom permissions stuff occurs now.

Reply:
Oh I see. My brain processed your earlier comment as only needing better naming

Reply:
Made the change and tested it. Question, is this the way you want to do it?
```ruby
  def need_to_simplify_email?(change_request)
    if GalleryConfig.email.force_simplified_emails
      return true
    end
    return false
  end
  ```
  And then have a custom method that overrides this? I only ask because to people looking in, we have parameters that are never used.